### PR TITLE
Added fundementals directive info

### PR DESCRIPTION
**Pull request by devlead**
Added a little info about directives, was hard to find info about them before.

### CODE REVIEW

File: src/Cake.Web/App_Data/docs.xml [hunk 1]
```diff
@@ -19,6 +19,7 @@
     <item title="Setup and teardown" file="fundamentals/setup-and-teardown.md" />
     <item title="Running targets" file="fundamentals/running-targets.md" />
     <item title="Script aliases" file="fundamentals/aliases.md" />
+    <item title="Directives" file="fundamentals/directives.md" />
```
Comment (patriksvensson) [inline src/Cake.Web/App_Data/docs.xml]: Maybe this should be called `Preprocessor directives` (and the file `preprocessor-directives.md`)

File: src/Cake.Web/App_Data/docs/fundamentals/directives.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+---
+content-type: markdown
+---
+
+Cake scripts have support for pre-processor line directives, which run before the script is executed.
```
Comment (patriksvensson) [inline src/Cake.Web/App_Data/docs/fundamentals/directives.md]: Should remove `line` here I think.

**Comment by patriksvensson** [comment]
@devlead Awesome! :+1:

**Comment by devlead** [comment]
Updated and rebased pr.

**Pull request merged**